numba/core/serialize.py

```diff
@@ -159,10 +160,80 @@ def _numba_unpickle(address, bytedata, hashed):
     return obj
 
 
-def dumps(obj):
+from collections import Mapping, Set, Sequence
+
+try:
+    from six import string_types, iteritems
+except ImportError:
+    string_types = (str, unicode) if str is bytes else (str, bytes)
+    iteritems = lambda mapping: getattr(mapping, 'iteritems', mapping.items)()
```

Comment:
Python 2 is no longer supported, right? So there's no need to do this.

Reply:
That's just some debugging code to find unpickable objects.  That will be removed once we have this working and don't need to track down pickling errors again.